addons/event/models/event_event.py

```diff
@@ -530,6 +527,22 @@ def write(self, vals):
             self.message_subscribe([vals['organizer_id']])
         return res
 
+    def name_get(self):
+        """Adds ticket seat availability if requested by context."""
+        if not self.env.context.get('name_with_seat_availability'):
+            return super().name_get()
+        res = []
+        for event in self:
+            # event or its tickets are sold out
+            if event.event_registrations_sold_out:
+                availability = _("Sold out")
+            elif event.seats_max:
+                availability = _('%(count)i seats remaining', count=event.seats_available)
```

Comment:
don't forget to format the count (with formatLang) exemple in French 1000 --> 1 000